Tasks/AppCenterDistributeV2/Tests/L0FailsHttpStatusNot2xx.ts

```diff
@@ -0,0 +1,199 @@
+
+import ma = require('vsts-task-lib/mock-answer');
```

Comment:
vsts-task-lib is deprecated at the moment - could you use https://github.com/microsoft/azure-pipelines-task-lib instead?

Reply:
Fixed in 37db5a37884ee302f50b6171b05de17089f64a2e